### DISCUSSION TITLE
PPOCRLabel报错：ppocr WARNING: When args.layout is false, args.ocr is automatically set to false

### DISCUSSION THREAD

**Discussion by aqingGitHub** [PaddlePaddle/PaddleOCR#13355] [Q&A] [answered]
1.打开PPOCRLabel报错如下，但仍然能打开软件：
(paddle2) D:\ProgramData\PaddleOCR275\PPOCRLabel>python PPOCRLabel.py --lang ch
[2024/07/12 13:43:24] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false

2.导入图片后报错如下：

file name in openNext is  D:\ProgramData\PaddleOCR275\train_data\C0441\1.jpg
DirPath in importDirImages is D:/ProgramData/PaddleOCR275/train_data/C0441
Using model from  paddle
can only concatenate str (not "float") to str
Traceback (most recent call last):
  File "D:\ProgramData\PaddleOCR275\PPOCRLabel\libs\autoDialog.py", line 58, in run
    strs += "Transcription: " + chars + " Probability: " + str(cond) + \
TypeError: can only concatenate str (not "float") to str

3.单击自动标注，但软件无任何进度反应，如下图
![image](https://github.com/user-attachments/assets/d0035049-b40f-4816-8024-d1607d787eec)

**Comment by aqingGitHub** [comment]
我把图片名字改成a、b、c字母后又报错如下：
(paddle2) D:\ProgramData\PaddleOCR275\PPOCRLabel>python PPOCRLabel.py --lang ch
[2024/07/12 13:54:29] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false
file name in openNext is  D:\ProgramData\PaddleOCR275\train_data\image\a.jpg
DirPath in importDirImages is D:/ProgramData/PaddleOCR275/train_data/image
Using model from  paddle
list index out of range
Traceback (most recent call last):
  File "D:\ProgramData\PaddleOCR275\PPOCRLabel\libs\autoDialog.py", line 43, in run
    self.result_dic = self.ocr.ocr(Imgpath, cls=True, det=True)[0]
IndexError: list index out of range

**Comment by GreatV** [comment]
参考 https://github.com/PFCCLab/PPOCRLabel/issues/19

**Reply by aqingGitHub** [reply, marked as answer]
安装paddleocr==2.6.1.3 可以解决问题, 谢谢啦！

**Comment by GreatV** [comment]
@aqingGitHub  后面PPOCRLabel的issue，可以提到 https://github.com/PFCCLab/PPOCRLabel/issues
